### ISSUE TITLE
Ошибка стр. 234

### ISSUE THREAD

**Issue by EvgenyRomanov** [igorsimdyanov/php8#258]
ошибка в примере с array_splice

![image](https://user-images.githubusercontent.com/43170380/219870189-eec52061-be9b-4acf-999f-9f1a55d4365e.png)

![image](https://user-images.githubusercontent.com/43170380/219870212-cf89267c-facc-4aa6-8fcb-95a6cede1d11.png)

**Comment by IrinaFosteeva** [comment]
Нет, в книге всё правильно, там выводится print_r(**$arr**), а ты в коде выводишь то, что вернул array_splice(), а не то, что лежит в $arr после выполнения array_splice() над ним. 

Но комменты в коде книги конечно логичнее было указать после строки с print_r(), чтобы не запутать никого.
